project.clj

```diff
@@ -39,6 +39,7 @@
                                org.apache.httpcomponents/httpclient
                                net.sourceforge.nekohtml/nekohtml
                                ring/ring-core]]
+                 [com.cemerick/url "0.1.1"]                           ; URL library
```

Comment:
That's definitely not the case, at least if you manually set the site-url in the admin settings.